android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -53,7 +54,13 @@ internal fun StoreProduct.mapProductCategory(): String {
 internal fun StoreProduct.mapProductType(): String {
     return when (type) {
         ProductType.INAPP -> "CONSUMABLE"
-        ProductType.SUBS -> "AUTO_RENEWABLE_SUBSCRIPTION" // TODO: Add a new string here prepaid (check recurrence mode)
+        ProductType.SUBS -> {
+            if (defaultOption?.fullPricePhase?.recurrenceMode == RecurrenceMode.NON_RECURRING) {
```

Comment:
i'm not remembering, do we consume this product type anywhere? just wanna be sure we remember to check for prepaid wherever we consume this too...

Reply:
It doesn't _look_ like it. I doesn't even look like its used in Flutter 🤷‍♂️